components/common/labels/List.tsx

```diff
@@ -97,7 +97,7 @@ export const ModerationLabel = ({
   const labelVal = toLabelVal(label, recordAuthorDid)
   const isSelfLabeled = isSelfLabel(labelVal)
   const labelDefFromService =
-    labelerServiceDef?.policies.definitionById[label.val]
+    labelerServiceDef?.policies.definitionById?.[label.val]
```

Comment:
This can crash a page if a plain label is added when the policy definition does not exist. So, this is more of a safety fallback.